package.json

```diff
@@ -80,6 +80,7 @@
     "numeral": "^2.0.4",
     "react": "^15.4.2",
     "react-relay": "https://github.com/alloy/relay/releases/download/v0.9.3/react-relay-0.9.3.tgz",
+    "react-styled-flexboxgrid": "^1.0.2",
```

Comment:
We need a grid that supports the responsive layout. When rendering on a mobile display, we need to drop two columns. That is not supported by the grid we have and would take some time to implement. That grid component fits right in as it's built on top styled-components and the API is virtually the same except you can specify column widths depending on the screen size.

```javascript
import React from 'react'

import {Grid, Col, Row} from 'react-styled-flexboxgrid'

const App = props =>
  <Grid>
    <Row>
      <Col xs={6} md={3}>Hello, world!</Col>
    </Row>
  </Grid>
```

![screen shot 2017-04-13 at 10 00 14 am](https://cloud.githubusercontent.com/assets/296775/25007912/0cc86d48-2030-11e7-9f78-203fac873692.png)